src/Keboola/StorageApi/Client.php

```diff
@@ -2207,6 +2207,29 @@ public function listFiles(ListFilesOptions $options = null)
      * @return int - created event id
      */
     public function createEvent(Event $event)
+    {
+        $result = $this->apiPostJson('events', [
```

Comment:
Sem i do createEventWithFormData prosím nad ten api call přidat komentáře `// DON'T FORGET to change createEventWithFormData as well` (a naopak). Případně vyextrahovat to pole do společné metody

Reply:
Půjde to automaticky přes extract methoda nabídne ti to i tu druhou jestli ji chceš taky replacnout ;)

Reply:
dobra pripominka, vyextrahovano